infra/osd-aws-pipeline.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: provision-osd-aws
+spec:
+  description: Provision OSD on AWS
+  params:
+    - default: kua-aws-credentials
+      description: Name of secret that contains credentials for osdCcsAdmin
+      name: aws-credentials
+      type: string
+    - default: kua-ocm-stage-client-credentials
+      description: Name of secret that contains client ID and client secret to log into OCM org
+      name: ocm-credentials
+      type: string
```

Comment:
I am thinking if it would not be better to use Secret object to store credentials like this. Firstly we could save credentials once in cluster and when running the pipeline we would just choose the cluster name and staging/prod env. Secondly the credentials would not be stored in PipelineRun mostly in plaintext - not a huge problem but maybe better practice.